jms-activemq-classic/src/main/resources/META-INF/native-image/io.micronaut.jms/jms-activemq-classic/reflect-config.json

```diff
@@ -0,0 +1,43 @@
+[
```

Comment:
I don't think this is correctly named. It would need to be `META-INF/native-image/io.micronaut.jms/micronaut-jms-activemq-classic/reflect-config.json` otherwise warnings will be produced in the native image maven plugin